lib/src/simple_speed_dial.dart

```diff
@@ -14,20 +15,43 @@ class SpeedDial extends StatefulWidget {
     this.openBackgroundColor,
   });
 
-  final Widget? child;
+  /// The speed dial display widget.
```

Comment:
It's the widget inside the main fab. This description isn't very clear.. what's a display widget? Please adjust or remove.